tests/roots/test-ext-autosummary-package/conf.py

```diff
@@ -0,0 +1,10 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('.'))
+
+extensions = ['sphinx.ext.autosummary']
+autosummary_generate = True
+
+# The suffix of source filenames.
+source_suffix = '.rst'
```

Comment:
Since 2.0, the default setting of `source_suffix` becomes `.rst`. So this setting is not needed.

Reply:
done